core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.kt

```diff
@@ -269,8 +271,8 @@ class STDCMPostProcessing(private val graph: STDCMGraph) {
                     TrainStop(
                         offset.meters,
                         timeData.stopTimeData[stopIndex++].currentDuration,
-                        // TODO: forward and use onStopSignal param from request
-                        isTimeStrictlyPositive(prevNode.stopDuration)
+                        // TODO: forward and use receptionSignal param from request
+                        if (isTimeStrictlyPositive(prevNode.stopDuration)) SHORT_SLIP_STOP else OPEN
```

Comment:
Same as the other comment, I don't know which is the most "generic" option for stops

Reply:
See https://github.com/OpenRailAssociation/osrd/pull/8983#discussion_r1774971041 (resolving this conversation for now).